### PR TITLE
/unreddit

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
BtaZ1PT7fL

## Why It's Good For The Game
Code that looks better and is less painful to read.
Smite panel that is less painful to use.

## Changelog
:cl:
code: Cosmetic improvements to admin smite code
admin: Some smites have been renamed.
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
booooo

**Comment by ATH1909** [comment]
cring

**Comment by Fikou** [comment]
b stands for b blood type, which is that those smites cause (blood)

**Comment by Ryll-Ryll** [comment]
Boooo this sucks

**Comment by Ryll-Ryll** [comment]
This also breaks up the grouping of those 3 smites since they're all related to each other

**Comment by Pumpkinoe** [comment]
boring

**Comment by Paxilmaniac** [comment]
Hardly a code improvement, I think the bot is lying to us.

**Comment by interestingusernam3** [comment]
>This also breaks up the grouping of those 3 smites since they're all related to each other

If that was the intention then why doesn't scarify start with a ":"?

**Comment by Ryll-Ryll** [comment]
> > This also breaks up the grouping of those 3 smites since they're all related to each other
> 
> If that was the intention then why doesn't scarify start with a ":"?

Because that doesn't actually hurt them

**Review by Cyberboss — APPROVED** [review]

File: code/modules/admin/smites/boneless.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /// Gives the target critically bad wounds
 /datum/smite/boneless
-	name = ":B:oneless"
+	name = "I'm going to saw through your bones"
```
Comment (cacogen) [inline code/modules/admin/smites/boneless.dm:3]:
```suggestion
	name = "Boneless"
```

**Comment by cacogen** [comment]
Nice PR title

Anyway this black Twitter meme ran its course years ago and epic xD jokes in the UI are insufferable

**Comment by Jared-Fogle** [comment]
have you confirmed the admins want this?

File: code/modules/admin/smites/boneless.dm (hunk 1 above)
Comment (interestingusernam3) [reply code/modules/admin/smites/boneless.dm:3]: Boneless is also reddit

**Comment by interestingusernam3** [comment]
>have you confirmed the admins want this?

No.

File: code/modules/admin/smites/boneless.dm (hunk 1 above)
Comment (Fikou) [inline code/modules/admin/smites/boneless.dm:3]: this name sucks and is even more reddit

Comment (cacogen) [reply code/modules/admin/smites/boneless.dm:3]: A TF2 quote is peak zoomer and this fits with the naming scheme and the typepath

**Comment by Timberpoes** [comment]
As a maintainer, this change only impacts admins and I would only want to push this forward to a merge if admins as a whole wanted it.

As an admin, I do not want this.

File: code/modules/admin/smites/boneless.dm (hunk 1 above)
Comment (interestingusernam3) [reply code/modules/admin/smites/boneless.dm]: Replaced.

Comment (interestingusernam3) [reply code/modules/admin/smites/boneless.dm]: Replaced.

**Comment by CoffeeDragon16** [comment]
🅱ooooo we like it, hopelessly outdated memes are soul, just look at gondolas

**Comment by NamelessFairy** [comment]
Personally I don't have an issue with the "de-redditfication" of the smite options but I'm not a fan of using "debone" over boneless, it just sounds a bit flimsy in comparison to "perforate", bloodless/boneless also suffered from this but not to the same extend. I'm aware that boneless and bloodless both look too similar at a glance so could I suggest "exsanguinate" over bloodless and  "fracture" over debone.

**Comment by cacogen** [comment]
Gondolas aren't a Twitter meme ran into the ground by normies

**Comment by Jared-Fogle** [comment]
The admins overwhelmingly said they didn't want this.

**Comment by cacogen** [comment]
👌🏿👌🏿👌🏿👌🏿

**Comment by interestingusernam3** [comment]
>The admins overwhelmingly said they didn't want this.

Redditor admin cabal confirmed!

**Pull request closed without merging**